### PR TITLE
show an error message in wagtail when there is no language

### PR DESCRIPTION
**Pull request by saeedmb8**

### CODE REVIEW

File: molo/core/cookiecutter/scaffold/{{cookiecutter.directory}}/{{cookiecutter.app_name}}/templates/admin/language_error_message.html [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{% raw %}
+{% load i18n wagtailadmin_tags staticfiles %}
+{% if error %}
+<div class="nice-padding panel-upgrade-notification" style="display:none">
+    <div class="help-block help-warning"><strong>Your site does not have a language.</strong> Click <a style="display: inline;"href="/admin/modeladmin/core/sitelanguage/">here</a> to add a language.</div>
```
Comment (saeedmb8) [inline molo/core/cookiecutter/scaffold/{{cookiecutter.directory}}/{{cookiecutter.app_name}}/templates/admin/language_error_message.html]: @miltontony are you happy with the error message?

**Comment by saeedmb8** [comment]
@miltontony ready for review :)

File: molo/core/cookiecutter/scaffold/{{cookiecutter.directory}}/{{cookiecutter.app_name}}/templates/wagtailadmin/home/site_summary.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{% raw %}
```
Comment (miltontony) [inline molo/core/cookiecutter/scaffold/{{cookiecutter.directory}}/{{cookiecutter.app_name}}/templates/wagtailadmin/home/site_summary.html]: Why do we need this file

File: molo/core/cookiecutter/scaffold/{{cookiecutter.directory}}/{{cookiecutter.app_name}}/templates/wagtailadmin/home/upgrade_notification.html [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{% raw %}
```
Comment (miltontony) [inline molo/core/cookiecutter/scaffold/{{cookiecutter.directory}}/{{cookiecutter.app_name}}/templates/wagtailadmin/home/upgrade_notification.html]: same here, we don't need this

File: molo/core/cookiecutter/scaffold/{{cookiecutter.directory}}/{{cookiecutter.app_name}}/templates/admin/language_error_message.html (hunk 1 above)
Comment (miltontony) [inline molo/core/cookiecutter/scaffold/{{cookiecutter.directory}}/{{cookiecutter.app_name}}/templates/admin/language_error_message.html]: :ok_hand:

**Comment by miltontony** [comment]
Looks good, just a question on some templates

File: molo/core/cookiecutter/scaffold/{{cookiecutter.directory}}/{{cookiecutter.app_name}}/templates/wagtailadmin/home/upgrade_notification.html (hunk 3 above)
Comment (saeedmb8) [inline molo/core/cookiecutter/scaffold/{{cookiecutter.directory}}/{{cookiecutter.app_name}}/templates/wagtailadmin/home/upgrade_notification.html]: @miltontony these templates are in the cookiecutters so they need the `{% raw %}` tag for scaffolding

File: molo/core/wagtail_hooks.py [hunk 4]
```diff
@@ -85,3 +85,28 @@ def get_context(self):
...
+        languages = SiteLanguage.objects.all()
+        if not languages:
+            return {
+                'error': True,
+            }
+
+
+@hooks.register('construct_homepage_panels')
+def add_language_error_message_panel(request, panels):
+    return panels.append(LanguageErrorMessage(request))
```
Comment (miltontony) [inline molo/core/wagtail_hooks.py]: Over here, you can do 
```python
if SiteLanguage.objects.all().exists():
    return [LanguageErrorMessage(request)]
return panels
```

**Comment by miltontony** [comment]
:+1: :cake:

**Pull request merged**